### ISSUE TITLE
value-keyword-case

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1027]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

New rule `unit-keywords-case`. Check case of units. Options: "lower"|"upper"

```css
a {
  display: block;
}
```

```css
a {
  display: BLOCK;
}
```

**Comment by davidtheclark** [comment]
Are both of those *valid*? Or are uppercase invalid?

**Comment by evilebottnawi** [comment]
@davidtheclark You can write in upper and lower case.

**Comment by jeddy3** [comment]
As with #1025, "[Keywords are CSS identifiers and are interpreted ASCII case-insensitively](https://www.w3.org/TR/css3-values/#keywords)".

How does the following sound?:

- Name: `value-keyword-case`
- Primary options: `"upper"|"lower"`
- Secondary options: none
- Message: `Expected "${actual}" to be "${expected}" (value-keyword-case)`.

**Comment by evilebottnawi** [comment]
@jeddy3 sound good

**Comment by evilebottnawi** [comment]
@jeddy3 WIP
